linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/template/router_container.template

```diff
@@ -2,6 +2,7 @@
   <div class="router router-{{this}}" data-router="{{this}}">
     <div class="summary"></div>
     <canvas class="router-graph" height="181"></canvas>
-    <div class='server router-servers clearfix'><div>Servers</div></div>
+    <div class="clients router-clients clearfix"><div>Clients</div></div>
+    <div class="servers router-servers clearfix"><div>Servers</div></div>
```

Comment:
I don't think clearfix is necessary anymore (may need the grid branch to land first? not sure)